fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -79,6 +79,9 @@
     private PendingCacheUpdate pendingCacheUpdate;
     private boolean maintainHistory = true;
 
+    @Nullable
+    private Descriptors.FileDescriptor localFileDescriptor;
```

Comment:
I think this would live more at home between the `dependencies` and the `extensionRegistry` member variable declarations given their parallel function. It definitely shouldn't have a blank line separating it from the other member variables.